### ISSUE TITLE
Incorrect confirmation message on deleting more than one category

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4730]
The confirmation toast message should be in correct plain english when deleting more than one Category.

https://github.com/ushahidi/platform/assets/8624777/4307dbd2-6bc0-4f28-94e2-1647e26ba1ca

---
Created automatically from USH-976 (https://linear.app/ushahidi/issue/USH-976/incorrect-confirmation-message-on-deleting-more-than-one-category)

**Comment by larrytamnjong** [comment]
Greetings @Angamanga my development environment is up and running can I be assigned to this issue 
![image](https://github.com/ushahidi/platform/assets/122117063/93ad76fd-0616-4930-a979-e4f401a51ad2)

**Comment by Angamanga** [comment]
@larrytamnjong Well done! You are welcome to go ahead on this issue!

**Comment by larrytamnjong** [comment]
Thanks @Angamanga :-)

**Comment by larrytamnjong** [comment]
Hello @Angamanga,
I've submitted a pull request to this minor issue, as outlined here: https://github.com/ushahidi/platform-client-mzima/pull/816. I would greatly appreciate your feedback on the implemented fix.

This particular issue is consistent across various sections; for instance, attempting to delete multiple Users, Posts, etc., would lead to the same problem. I haven't addressed these yet, as I wanted to confirm if the fix in https://github.com/ushahidi/platform-client-mzima/pull/816 is satisfactory. Once confirmed, I'll go ahead and implement the fix in other sections as well.

Thank you.
